### ISSUE TITLE
CTD during character creation when selecting a specific head style: 018Auri.bsp (018AuriHead) (Song of the green)

### ISSUE THREAD

**Issue by drmattsuu** [ForgottenGlory/Living-Skyrim-2#477]
Crash to Desktop Log: [Crash_2020_10_13_18-57-31.txt](https://github.com/ForgottenGlory/Living-Skyrim-2/files/5373273/Crash_2020_10_13_18-57-31.txt) 

**LS Version**
Living Skyrim v2.5.0

**Describe the bug**
Skyrim crashes to desktop during character creation if the user selects a specific head-style for their character, see repro steps.

**To Reproduce**
- Start a new game
- Select the female body type
- Select the Breton race (this race produces the most reliable crash)
- Tab to the "Head" section
- On the "Face Part" opton, select the option that corrisponds with 018Auri.bsp (018AuriHead)

**Expected behavior**
- No crash is expected.

**Screenshots**
Highlighted option that will cause CTD
![image](https://user-images.githubusercontent.com/3444149/95900438-cfa04580-0d89-11eb-9dfc-7cced8687d78.png)

**Additional context**
This is likely an issue where the head mesh simply isn't compatible with the race, (looking at it with certain other races is slightly more stable but the mesh itself doesn't jive at all).

A simple work-around for this is simply don't use that head-mesh and avoid adjusting the option to the last two options on the slider to avoid the crash.  It's also likely a problem with the mod rather than a problem with this mod-list, but I wanted to post this for anyone else who might be searching for the issue.

**Comment by ForgottenGlory** [comment]
Known issue with that face part. Also, known fix: don't pick the face part. Easy. :)

**Comment by celadoneiron** [comment]
If the "playable" flag is removed from the head mesh using SSEEdit, it's no longer selectable in racemenu and so can't cause a crash. As far as I've been able to tell this doesn't cause any issues with Auri in-game, and it means that an accidental mis-click or keypress won't result in a CTD. I reported this on the mod's Nexus page as well, but figured I'd mention it here in case people who need to use that slider want to do it themselves.
